views/about.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/public/styles.css">
+    <link rel="icon" href="/public/favicon.ico">
+    <title>About</title>
+</head>
+<body>
+    <div class="nav">
+       <a ref="contacts.html" href="./contacts.html">Contacts</a>
```

Comment:
On the `href`, you don't want to send the user for your file. You want to redirect them to `/contacts` or `/home`. Pages (views) wich you are handling on your `app.js`. 
So, it needs to be like this:
`<a ref="contacts.html" href="/contacts">Contacts</a>`